basicLibrary/src/main/java/basicLibrary/Library.java

```diff
@@ -48,8 +48,7 @@ public static double getAverage(int[] arr) {
         for (int i = 0; i < arr.length; i++) {
             total += arr[i];
         }
-        return total / arr.length;
-    }
+        return total / arr.length;    }
```

Comment:
I recommend moving closing `}`'s to be on their own lines so that it's clear where the end of the function is